sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1447,6 +1447,24 @@ export class BlobClient extends StorageClient {
 
   // High level function
 
+  /**
+   * ONLY AVAILABLE IN NODE.JS RUNTIME.
+   *
+   * Downloads an Azure Blob in parallel to a buffer.
+   * Offset and count are optional, pass 0 for both to download the entire blob.
```

Comment:
If no offset & count are provided, then the entire blob is anyway downloaded right? Then why do we want to document the special case of passing 0?

Reply:
Same question

Reply:
Fixed the comment.